Tests/js/views/services/assets/ez-locationviewviewservice-tests.js

```diff
@@ -1368,10 +1368,520 @@ YUI.add('ez-locationviewviewservice-tests', function (Y) {
+                    } else {
+                        Y.fail("Unexpected parameter for content mock");
+                    }
+
+                }
+            });
+
+            Mock.expect(parentLocation, {
+                method: 'get',
+                args: ['id'],
+                returns: 'alan-shearer'
+            });
+
+            Mock.expect(this.location, {
+                method: 'deleteContent',
+                args: [Mock.Value.Object, Mock.Value.Function],
+                run: function (options, callback){
+                    callback(true);
+                }
+            });
+
+            Mock.expect(content, {
+                method: 'delete',
+                args: [Mock.Value.Object, Mock.Value.Function],
+                run: function (options, callback){
+                    callback(true);
+                }
+            });
+
+            this.service.on('confirmBoxOpen', function (e) {
+                e.config.confirmHandler.apply(this);
+            });
+
+            this.service.once('notify', function (e) {
+                this.once('notify', function (e) {
+                    notified = true;
+
+                    Assert.isObject(e.notification, "The event facade should provide a notification config");
+                    Assert.areEqual(
+                        "error", e.notification.state,
+                        "The notification state should be 'error'"
+                    );
+                    Assert.isString(
+                        e.notification.text,
+                        "The notification text should be a string"
+                    );
+                    Assert.isTrue(
+                        e.notification.text.indexOf(contentName) !== -1,
+                        "The notification text should contain content name"
+                    );
+                    Assert.isTrue(
+                        e.notification.identifier.indexOf(contentId) !== -1,
+                        "The notification identifier should contain content id"
+                    );
+                    Assert.areSame(
+                        0, e.notification.timeout,
+                        "The notification timeout should be set to 0"
+                    );
+                });
+            });
+
+            this.service.fire('whatever:deleteContentAction', {content: content});
+            Assert.isTrue(notified, "The notified event should have been fired");
+        },
+
+        "Should NOT fire deletedContent event when deleting content item failed": function () {
+            var content = new Mock(),
+                parentLocation = new Mock(),
+                that = this,
+                contentName = 'pierlugi-collina',
+                contentId = 'aContentId',
+                eventFired = false;
+
+            this.service.set('path', [{location: parentLocation}]);
+            this.service.set('content', content);
+
+            Mock.expect(content, {
```

Comment:
this mock configuration and the one for `parentLocation` are the same and repeated in every test, you should put that in one or 2 dedicated methods.